evaluation.md

```diff
@@ -1,19 +1,30 @@
 # Evaluation: running commentary
 
-## Internal DSL
+## External DSL
 
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
-**On a scale of 1–10 (where 10 is "a lot"), how much did you have to change your syntax?**
+I originally wanted it such that you could have a header which would represent a State, and then the rules after it. See:
 
-**On a scale of 1–10 (where 10 is "very difficult"), how difficult was it to map your syntax to the provided API?**
+State 1:
+if [N: open] -> go N, change to State 1
+if [N: open, S: blocked] -> go S, don't change State 
 
-## External DSL
+However, I originally found it difficult to implement this in my parser because I wasn't sure how to recognize each block as one state, and pass it to the Picobot API. Thus, I changed it to be:
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+From State 1, if [N: open] -> go N, change to State 1
+From State 1, if [N: open, S: blocked] -> go S, don't change State 
+
+Now that I'm a bit more familiar with how parsing works, though, I think I could potentially implement this; but alas, time. 
+
+I also wanted to have key words "stay here" and "don't change State" but again, I was spending a lot of time on this already so I decided to just replace it with "go Nowhere" and "change to State X" where X is the original state. This simplified the parsing code because they have the same overall template. Again, I think I could definitely change this in the future if I wanted to, though.
 
```

Comment:
I like this example of your code. I think it is quite simple and easy for a user to understand. I would totally love to see how you would change this to match your original vision in the future.